cpp/include/cudf/io_functions.h

```diff
@@ -43,3 +43,8 @@ gdf_error read_parquet(pq_read_arg *args);
  * @brief Interface to convert GDF Columns to Compressed Sparse Row
  */
 gdf_error gdf_to_csr(gdf_column **gdfData, int num_cols, csr_gdf *csrReturn);
+
+/*
+ * @brief Interface to parse JSON data to GDF columns
```

Comment:
I think we want to move the doxygen documentation to the interface header file here, like `write_csv`. We did it in the ORC PR as well. We can do it for the other interfaces in another PR.